DAL/BaseDao.cs

```diff
@@ -1,28 +1,21 @@
 ﻿using System.Configuration;
-using MongoDB.Bson.Serialization.Attributes;
-using MongoDB.Bson;
+using Model;
 using MongoDB.Driver;
 
 namespace DAL
 {
     public class BaseDao
     {
-        string connectionString;
-        MongoClient client;
+        protected readonly IMongoCollection<Employee> _employeeCollection;
```

Comment:
Please add this comment, so we will remember it's only references and not cached data, whuch is why it's readonly.

`// References to the collection in the MongoDB instance`